order-summary/run.js

```diff
@@ -1,27 +1,29 @@
+/**
+ *  @fileOverview Parses Shopify order webhook response JSON and converts to Flow Order Summary
+ *
+ *  @author       Matt Kersner
+ *  @author       John Bell
+ *
+ *  @requires     Node 11+
+ *  @requires     Flow API key
+ */
+
 const fs = require('fs');
 const https = require('https');
-var path = require('path');
+const path = require('path');
 
-// Loop through all the files in the temp directory
-fs.readdir("./samples", function (err, files) {
-  if (err) {
-    console.error("Could not list the directory.", err);
-    process.exit(1);
-  }
+/** "Samples" is the directory containing the input JSON files. Directory read from can be changed here. */
+const orderFilesDirectory = path.resolve(__dirname, 'samples');
 
-  files.forEach(async function (file, index) {
-    console.log('FILE ===>', file, index);
-    let rawdata = fs.readFileSync(`./samples/${file}`);
-    let order = JSON.parse(rawdata);
-    const completedOrder = await parseOrder(order)
-    let data = JSON.stringify(completedOrder, fetchImgFromFlow());
-    fs.writeFileSync(`./output/${file}`, data);
-  });
-});
 
+/**
+ * Takes a variant_id and returns an object from Flow's API
```

Comment:
`Takes a variant_id and returns an object from Flow's API` ->

```
Takes a Shopify variant ID and returns a a Flow item. This item is used to retrieve additional information such as images, which is required for the Order Summary object. We provide an implementation which retrieves this data from api.flow.io, however it can be customized to suit your needs. The only requirement is that it returns an `Item` model from Flow.
```

Also this function name should change to something like `getFlowItemFromVariant`